Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -3414,6 +3419,12 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
     HashSet<WebCore::FrameIdentifier> m_framesWithSubresourceLoadingForPageLoadTiming;
     RunLoop::Timer m_generatePageLoadTimingTimer;
 
+#if PLATFORM(COCOA)
+    RunLoop::Timer m_textIndicatorTimer;
```

Comment:
timer name should specify what it is about. this is not the text indicator timer, it's the text indicator ... something ... timer. Dismissal? fade? what is it?